googlemock/include/gmock/gmock-spec-builders.h

```diff
@@ -1464,6 +1464,9 @@ class FunctionMockerBase : public UntypedFunctionMockerBase {
   typedef typename Function<F>::ArgumentMatcherTuple ArgumentMatcherTuple;
 
   FunctionMockerBase() : current_spec_(this) {}
+#if GTEST_LANG_CXX11
+  FunctionMockerBase(FunctionMockerBase&&) = default;
```

Comment:
Visual Studio 2013 does not support defaulted move constructor/operator.

Reply:
There is no feature testing macro. So we should exclude Visual Studio 2013?

Reply:
It would be easier to implement it instead of using "= default" I think.

Reply:
`GTEST_LANG_CXX11` is not defined for Visual Studio 2013, so it's ok